src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderCharge.java

```diff
@@ -149,6 +149,7 @@ public OffenderOffence getOffenceDetail() {
             .offenderChargeId(id)
             .offenceStartDate(dateOfOffence)
             .offenceEndDate(endDate)
+            .offenceStatute(offence.getStatute().getCode())
```

Comment:
can this ever be null?

Reply:
No, the statute forms part of the offence primary key.